packages/firebase_crashlytics/test/firebase_crashlytics_test.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('$Crashlytics', () {
+    const MethodChannel channel =
+        MethodChannel('plugins.flutter.io/firebase_crashlytics');
+
+    final List<MethodCall> log = <MethodCall>[];
+
+    final Crashlytics crashlytics = Crashlytics.instance;
+
+    setUp(() async {
+      channel.setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+        switch (methodCall.method) {
+          case 'Crashlytics#isDebuggable':
+            return true;
+          case 'Crashlytics#setUserEmail':
+            return true;
+          case 'Crashlytics#setUserIdentifier':
+            return true;
+          case 'Crashlytics#setUserName':
+            return true;
+          case 'Crashlytics#log':
+            return null;
+        }
+      });
+      log.clear();
+    });
+
+    test('isDebuggable', () async {
+      expect(await crashlytics.isDebuggable(), true);
+      expect(log,
+          <Matcher>[isMethodCall('Crashlytics#isDebuggable', arguments: null)]);
+    });
+
+    test('log', () {
+      crashlytics.log('foo');
+      expect(crashlytics.logs.length, 1);
+      crashlytics.log('bar');
+      expect(crashlytics.logs.length, 2);
+    });
+
+    test('isInDebugMode', () {
```

Comment:
I don't think this needs to be tested. It shouldn't be part of the public API

Reply:
Done